hilbert_hotel/8.2.hilbert_hotel.py

```diff
@@ -0,0 +1,111 @@
+import random
+from hotel_room import HotelRoom
+
+
+class HotelBuilder:
+    """
+          A class that builds a hotel.
+          (Helper Class for HilbertHotel Class)
+          Attributes:
+              None
+
+          Methods:
+              build_hotel: Builds a hotel with the specified number of floors and total rooms.
+                           There are (total rooms // floors) per floor, so total rooms may decrease.
+              generate_room: Generates a HotelRoom object with a random room type, a floor, and room number.
+      """
+
+    @staticmethod
+    def build_hotel(floors, total_rooms):
+        hotel = []
+        rooms_per_floor = total_rooms // floors
+        for floor in range(1, floors + 1):
+            for room in range(rooms_per_floor):
+                hotel.append(HotelBuilder.generate_room(floor * 100 + room, floor))
```

Comment:
Replace these two for loops with a list comprehention with two iterators - 
https://stackoverflow.com/questions/18551458/how-to-frame-two-for-loops-in-list-comprehension-python